styles.css

```diff
@@ -36,6 +36,13 @@ img {
 
 @media(max-width: 600px) {
   .main {
-    background: red;
+    header{order:1;}
+    main{order:2;}
+    aside{order:3;}
+    aside{order:4;}
+    footer{order:5;}
   }
 }
+header, footer, aside, .main > {
+  margin: 0px;
+}
```

Comment:
Very close. You were correct in using `>` but it needs to be followed by something. In this case, you can use `*` so that this styling is applied to **all** direct children of those four selectors.

```css
header, footer, aside, .main > * {
  margin: 0px;
}
```